Week02/lesson04/homework/relaxr_starter_code/css/style.css

```diff
@@ -41,44 +42,150 @@ Developer @GA in NYC
   padding: 0 10px 10px 0; /* note the padding around a left floated image */
 }
 
+@font-face {
+    font-family: Open Sans;
+    src: url(https://www.google.com/fonts#UsePlace:use/Collection:Open+Sans);
+    font-weight: light, light italic, semibold, extrabold;
+}
 /******************************************
 /* BASE STYLES
 /*******************************************/
 
 body {
-     color: #000;
-     font-size: 12px;
-     line-height: 1.4;
-     font-family: Helvetica, Arial, sans-serif;
+     color: #f0efef;
+     font-family: Open Sans, Helvetica, Arial, sans-serif;
      background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
-      background-size: cover;
+     background-size: cover;
+}
+
+header {
+       font-size: 24px;
+       height: 500px;
+}
+
+h1 {
+   font-weight: extrabold;
+   margin: 75px 0 0 100px;
+}
+
+h2 {
+   font-weight: semibold;
+   text-align: center;
+   line-height: 250px;
+}
+
+nav{
+   background-color: #f9e42e;
+   width: 200px;
+   margin: 0 auto;
+   border-radius: 5px;
+}
+
+nav a {
+       font-weight: extrabold;
+       font-size: 14px;
+       color: #033048;
+       text-decoration: none;
+       line-height: 50px;
+       margin-left: 65px;
+}
+
+.benefits {
+          font-weight: light;
+          font-size: 13px;
+          color: #606161;
+          background-color: #f0efef;
+          height: 250px;
+}
+
+h3{
+  font-weight: semibold;
+  font-size: 20px;
+  color: #121212;
+  padding-top: 20px;
+  margin-left: 100px;
+}
+
+p{
+  padding-bottom: 20px;
+  margin-left: 100px
+}
+
+ol{
+   list-style-type: disc;
+   padding-bottom: 20px;
+   margin-left: 150px;
+   line-height: 2;
+}
+
+.quote {
+       height:130px;
+       background: url(../images/testimonial_bg.jpg);
 }
 
+blockquote {
+           font-style: italic;
+           font-weight: light italic;
+           font-size: 18px;
+           text-align: center;
+           padding: 20px 0 20px 0;
+}
+
+span {
+     font-style: normal;
+     font-weight: semibold;
+}
+
+.link {
+      height: 100px;
+      padding: 25px 0 25px 0;
+      background-color: #f9e42e;
+}
+
+.link nav {
+          background-color: #033048;
+}
+
+.link nav a {
+            margin-left: 55px;
+            color: #f9e42e;
+}
 
 /******************************************
 /* LAYOUT
 /*******************************************/
 
 /* Center the container */
+
 #container {
   width: 100%;
   margin: auto;
   height: 500px;
```

Comment:
So this was a bit of a trick by me to see if you guys would catch it (SO mean, I know! 🙈) but basically, if you use `#conatiner` as the div that surrounds all the content (which you did) then you want to remove this line, because you don't ever want to limit the height of all the content on the page (especially not to just 500px). Not that you can tell visually (as the content will break past the limitations set on its parent, at least in Chrome) but it's just a bad idea.

Instead, you want to apply the `height` limit to the `<header>` and apply the `background: url(../images/testimonial_bg.jpg);` there (not on the `body`). You only apply a full-width `background` to the body if you want the background to be static:

For example if you go here: https://css-tricks.com/examples/FullPageBackgroundImage/progressive.php you'll see that the page's background image never moves or changes, the code shows us the background image has been added to the `html` tag:

![image](https://cloud.githubusercontent.com/assets/5670279/17310082/21282c4c-583a-11e6-99f6-884db8c7cf91.png)

.... but this "never-changing" image is NOT the effect we want for Relaxr, so best to only apply the bg image to the header. And then limit the height of the header to get the desired effect.